students/wooseok-choi/user/views.py

```diff
@@ -1 +1,31 @@
+import json
 
+from django.http  import JsonResponse
+from django.views import View
+
+from user.models import User
+from user.validate import validate_email, validate_password
+
+
+class SignupView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        try:
+            if not validate_email(data['email']):
+                return JsonResponse({'message':'INVALID EMAIL'}, status=400)
+            if not validate_password(data['password']):
+                return JsonResponse({'message':'INVALID PASSWORD'}, status=400) 
+            if User.objects.filter(email = data['email']).exists():
+                return JsonResponse({'message':'DUPLICATE EMAIL'}, status=409)
```

Comment:
```suggestion
            if not validate_email(data['email']):
                return JsonResponse({'message':'INVALID EMAIL'}, status=400)
                
            if not validate_password(data['password']):
                return JsonResponse({'message':'INVALID PASSWORD'}, status=400)
                 
            if User.objects.filter(email = data['email']).exists():
                return JsonResponse({'message':'DUPLICATE EMAIL'}, status=409)
```
가독성 측면에서 다음과 같이 공백을 추가하면 좋을 것 같습니다.

Reply:
넵!